eureka-core/src/main/java/com/netflix/eureka/util/EurekaMonitors.java

```diff
@@ -89,9 +92,11 @@ private EurekaMonitors(String name, String description) {
 
     @com.netflix.servo.annotations.Monitor(name = "count", type = DataSourceType.COUNTER)
```

Comment:
Are the servo stats still needed? Internally this will cause duplication with both getting sent into Atlas.